projects/js-packages/social-logos/tools/svg-to-sprite.js

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+const svgDir = 'build/svg-clean';
+const srcSpriteDir = 'src/svg-sprite';
+const srcCssDir = 'src/css';
+const destSpriteDir = 'build/svg-sprite';
+const destSpriteFilename = `${ destSpriteDir }/social-logos.svg`;
+
+const fs = require( 'fs' );
+const path = require( 'path' );
+const process = require( 'process' );
+const { glob } = require( 'glob' );
+const svgstore = require( 'svgstore' );
+
+// Start in the right folder.
+const rootDir = __dirname + '/..';
+process.chdir( rootDir );
+
+// Make dir if it doesn't exist.
+if ( ! fs.existsSync( destSpriteDir ) ) {
+	fs.mkdirSync( destSpriteDir, { recursive: true } );
+}
+
+// Generate SVG.
+const sprites = svgstore( { inline: true } );
+const files = glob.sync( svgDir + '/*.svg' ).sort();
+for ( const file of files ) {
+	sprites.add( path.basename( file, '.svg' ), fs.readFileSync( file, 'utf8' ) );
+}
+
+const svgText = sprites
+	.toString()
+	// Make SVG more conformant.
+	.replace( '<svg><defs/>', '<svg xmlns="http://www.w3.org/2000/svg">' );
+
+fs.writeFileSync( destSpriteFilename, svgText, 'utf8' );
+
+// Copy example files.
+fs.cpSync( `${ srcSpriteDir }/.`, `${ destSpriteDir }/.`, { recursive: true } );
```

Comment:
Now there's only one file in here (`example.html`), so perhaps we should just explicitly move that. No need for recursive either.

```suggestion
fs.cpSync( `${ srcSpriteDir }/example.html`, `${ destSpriteDir }/example.html` );
```

Reply:
Meh. I was matching the sense of what the old code did (without having to manually do the glob and copy file by file).

Reply:
Yeah, the old code was an artifact of an older structure and I should've tightened it up at that point.